doc/website/getting-started/overview.md

```diff
@@ -11,637 +10,242 @@ runtime and create _publishers_ and _subscribers_. The publishers send data of a
 received by subscribers of the same topic. To enable publishers to offer their topic and subscribers to subscribe to
 these offered topics, the middleware daemon, called ``RouDi``, must be running.
 
-For further information how iceoryx can be used see the [examples](../../../iceoryx_examples/README.md). The
-[conceptual guide](../../conceptual-guide.md) provides additional information about the _Shared Memory
-communication_ that lies at the heart of iceoryx.
+<!-- @todo add overview graphic -->
 
-We now briefly define the main entities of an iceoryx system before showing how they are created and used by the
-iceoryx API.
+For further information how iceoryx can be used, see the 
+[examples](https://github.com/eclipse-iceoryx/iceoryx/blob/master/iceoryx_examples/README.md). The
```

Comment:
```suggestion
[examples](../examples/). The
```
I wouldn't jump to GitHub here, this could be confusing for readers. There will be a link to the code at the end of each example.

Reply:
I would jump to GitHub here since most of the developers will use the GitHub documentation.
I have the feeling the thing with the website is somehow getting out of hand. Instead of optimizing markdown documents for a website, we should go the clean way and add markdown comments and fill them out with a bash script so that, when the website is generated everything looks clean.

This is getting weirder and uncleaner ...

Quality over quantity! The website is also not mandatory till Thursday but a clean docu is!